build-logic/src/main/kotlin/org/terasology/gradology/module_deps.kt

```diff
@@ -90,13 +90,10 @@ fun moduleDependencyOrdering(modulesConfig: Configuration): List<String> {
     // configurations.resolvedConfiguration is more straightforward if you just want all the artifacts,
     // but using `.incoming` lets us turn on lenient mode as well as do more accurate filtering of local modules
     val resolvable = modulesConfig.incoming
-    val artifactView = resolvable.artifactView {
-        lenient(true)
-    }
```

Comment:
Is this no longer needed or no longer possible?

Reply:
[Docs for artifactView](https://docs.gradle.org/current/javadoc/org/gradle/api/artifacts/ResolvableDependencies.html#artifactView-org.gradle.api.Action-l)

Reply:
💡